.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -31,6 +31,12 @@ body:
       description: A clear description of what you expected to happen.
     validations:
       required: true
+  - type: textarea
+    attributes:
+      label: System Info
+      description: Output of `npx envinfo --system --npmPackages @stencil/core --binaries --browsers`
+      render: shell
+      placeholder: System, Binaries, Browsers
```

"Binaries" might be a little too specific. Maybe "Environment"?
```suggestion
      placeholder: System, Environment, Browsers
```